Resources/doc/installation.rst

```diff
@@ -78,6 +78,19 @@ the MongoDB ODM across your application:
     Of course, you'll also need to make sure that the MongoDB server is running
     in the background. For more details, see the MongoDB `Quick Start`_ guide.
 
+
+###Authorization
```

Comment:
```suggestion
Configuration
-------------
```

This is a RST file so this is the correct header

Reply:
I guess `Configuration` in your suggestion should be replaced with `Authorization`

Reply:
Right! Sorry, it's too late :)